databases/week2/week2exercise1.sql

```diff
@@ -0,0 +1,45 @@
+-- Add a task with the these attributes: title, description, created, updated, dueDate, statusID, userID
+INSERT INTO task (
+title,
+description, 
+created, 
+updated, 
+due_date, 
+status_id, 
+user_id
+) VALUES (
+"Learn about MySQL",
+"interesting",
+NOW(),
+NOW(),
+"2019-12-23 10:13:42",
+1,
+7
+);
```

Comment:
I would recommend that you indent the column names and the values just like you would inside a javascript function. 

Also while double quotes work for strings in MySQL they wont work in other databases. The standard way of specifying strings is using single quotes.

Reply:
I updated the file with single quotes.